elasticsearch/kibana/map.jinja

```diff
@@ -22,3 +15,31 @@
         'ssl': {}
     },
 }, grain='os_family', merge=salt['pillar.get']('kibana:lookup'), base='default') %}
+
+{% if elasticsearch.elastic_stack %}
+{% set kibana_conf = {
+    'config': {
+            'elasticsearch.url': 'http://localhost:9200',
+            'server.host': '127.0.0.1',
+            'server.port': 5601
+        },
+    'nginx_config': {
+            'server.ssl.certificate': '/etc/salt/ssl/certs/kibana.yourdomain.com.crt',
```

Comment:
It seems unnecessary to rename the keys for the Nginx config since they are not referenced directly by Kibana.